lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -729,7 +726,7 @@ def main():
     host_default_vnc_port_start = module.params['host_default_vnc_port_start']
     host_default_vnc_port_end = module.params['host_default_vnc_port_end']
     subscription = module.params['subscription']
-    uid_ems = module.params['uid_ems']
+    uid_ems = module.params['azure_tenant_id']
```

Comment:
You're still sending this to API as uid_ems, this means this PR is not dependant on the API PRs to allow `azure_tentant_id`, right?

That's cool, I just want to understand the plan for ansible X manageiq compatibility.
1. will uid_ems always work or will module *have* to send `azure_tenant_id` later?
2. how long has manageiq already supported `uid_ems` for this?  any stable releases?
  I.e. if module later switches to sending `azure_tenant_id` will it break older manageiq and will it need to try both?
  (there's no way to detect which version we talk to, except by trying and catching exception?)
3. This module already accepted `uid_ems` param — for just a few days since #34708 — is it OK to remove it now?
  "Feels OK" to me but @yaacov @elad661 do you know what are the ansible rules for this?
  (I know, I reviewed #34708, sorry didn't read the comments)

Personally, I'm cool with dropping `uid_ems` baggage completely, but that might depend on (2) and (3).

Reply:
regarding question number 3, I think it's okay to remove it - this has only been merged for 3 days, and wasn't in any released version of ansible (afaik), so it's unlikely that anyone is relying on it anyway.